### ISSUE TITLE
When I training RNet, the all of loss are nan?

### ISSUE THREAD

**Issue by tjusxh** [AITTSMD/MTCNN-Tensorflow#106]
When I training PNet, the all of loss are normal.
2018-05-09 07:18:59.984939 : Step: 117700, accuracy: 0.956863, cls loss: 0.212693, bbox loss: 0.068303, landmark loss: 0.558273,L2 loss: 0.036567,lr:0.000010 
2018-05-09 07:19:14.511240 : Step: 117800, accuracy: 0.917323, cls loss: 0.293987, bbox loss: 0.063860, landmark loss: 0.750174,L2 loss: 0.036567,lr:0.000010 
2018-05-09 07:19:29.180299 : Step: 117900, accuracy: 0.907950, cls loss: 0.311420, bbox loss: 0.067427, landmark loss: 0.727518,L2 loss: 0.036567,lr:0.000010 

**When I train RNet, the all of loss are nan, anyone know the reason?**
2018-05-10 13:17:09.375676 : Step: 14000, accuracy: 0.750000, cls loss:  nan, bbox loss:  nan, landmark loss:  nan,L2 loss:  nan,lr:0.010000 
2018-05-10 13:19:28.685029 : Step: 15000, accuracy: 0.750000, cls loss:  nan, bbox loss:  nan, landmark loss:  nan,L2 loss:  nan,lr:0.010000 
2018-05-10 13:21:47.784554 : Step: 16000, accuracy: 0.750000, cls loss:  nan, bbox loss:  nan, landmark loss:  nan,L2 loss:  nan,lr:0.001000

**Comment by AITTSMD** [comment]
@tjusxh The training data for RNet is likely  wrong.Check your training data and you had better visual it.

**Comment by darengking** [comment]
@tjusxh I have met the same problem. Have you solved it yet?
